src/data-visualisation/accessibility/design-a11y.md

```diff
@@ -0,0 +1,107 @@
+  
+::: details Source
+ 
+[WCAG 2.1 Use of Colour][use of colour]
+
+:::
+
+
+
+## Designing charts for accessibility  
+  
+Charts can often be confusing to a user, and enabling a user to understand the content shown is one of our primary purposes in development.  
+
+Some best practices for designing charts accessibly:  
+- Declutter where possible, but do not remove any useful content. Content should be efficient and effective
+* Remove gridlines 
+* Minimum colours
+* Don’t have a busy chart  
+- Include alt text (for people with low vision or those who find charts confusing)
+- Include the source of your data, to allow users to familiarise themselves with the data and aid data literacy
+- No wonky text, horizontal is best
+- Don’t use italics, this can be hard for a user to read
+- Make sure your product adapts for smaller screens. This is another reason why a text alternative for a chart is recommended
+- Sans serif font is more understandable for people with dyslexia. Make sure your chart text is as large as is reasonable
+- Don’t mislead the user, for example, start a chart’s y axis from 0 can help ensure the user understands the data clearly
+- Choosing the right type of chart, to aid a user’s understanding
+- Be aware of where your content is placed. Chart descriptions should appear above a chart to allow a screen reader to pick up the information before a chart is shown
+- Label all buttons, hyperlinks, and necessary features accurately
+- If a hyperlink is included, hyperlink appropriate text describing the hyperlink; this is suitable for a screen reader to pick up
+
+## Spreadsheet accessibility  
+  
+There is a checklist to follow online which the Government Analysis Function created.  
+The following links may also be useful:  
+- [Making spreadsheets accessible: a checklist of the basics – Government Analysis Function][gov 1]
+- [Releasing Statistics in Spreadsheets][gov 2] 
+- [Make your Excel documents accessible to people with disabilities - Microsoft Support][microsoft]  
+  
+    
+[Back to Accessibility home](../a11y/)
+
+
+
+[home office posters]: https://github.com/UKHomeOffice/posters/blob/master/accessibility/dos-donts/posters_en-UK/accessibility-posters-set.pdf
+[gov 1]: https://analysisfunction.civilservice.gov.uk/policy-store/making-spreadsheets-accessible-a-brief-checklist-of-the-basics/
+[gov 2]: https://analysisfunction.civilservice.gov.uk/policy-store/releasing-statistics-in-spreadsheets/
+[microsoft]: https://support.microsoft.com/en-us/office/make-your-excel-documents-accessible-to-people-with-disabilities-6cc05fc5-1314-48b5-8eb3-683e49b3e593
+[numbers 1]: https://nhsbsauk.sharepoint.com/sites/DigitalContentDesignTeam/SitePages/NHSBSA-digital-style-guide-and-standards.aspx
```

Comment:
Link seems to be pointing to an internal site. Don't think we want to include these @yantantether ?

Reply:
I have added a caveat where this link appears on the page.